src/redux/actions/auth.action.js

```diff
@@ -39,3 +39,12 @@ export const login = () => async (dispatch) => {
     });
   }
 };
+
+export const logout = () => async (dispatch) => {
+  await auth.signOut;
```

Comment:
Is this not a function?

Reply:
yes I saw it now, but I checked my app and its working fine without these paranthesis @faran4engg